kraken/pod_scenarios/setup.py

```diff
@@ -59,18 +59,13 @@ def container_run(kubeconfig_path, scenarios_list, config, failed_post_scenarios
         with open(container_scenario_config[0], "r") as f:
             cont_scenario_config = yaml.full_load(f)
             for cont_scenario in cont_scenario_config["scenarios"]:
-                if len(container_scenario_config) > 1:
-                    pre_action_output = post_actions.run(kubeconfig_path, container_scenario_config[1])
-                else:
-                    pre_action_output = ""
                 # capture start time
                 start_time = int(time.time())
-                container_killing_in_pod(cont_scenario)
+                killed_containers = container_killing_in_pod(cont_scenario)
                 logging.info("Waiting for the specified duration: %s" % (wait_duration))
                 time.sleep(wait_duration)
-                failed_post_scenarios = post_actions.check_recovery(
-                    kubeconfig_path, container_scenario_config, failed_post_scenarios, pre_action_output
-                )
+                failed_post_scenarios = check_failed_containers(killed_containers, cont_scenario.get("retry_wait", 120))
```

Comment:
Maybe move this check to line 65? This way the scenario waits for the container to recover and then waits for the usual time specified in the config. Thoughts?

Reply:
In pod scenarios we do the wait duration first and then do the check post actions. I could see it being helpful both ways. I could change either to match. definitely think we should be consistent